### ISSUE TITLE
Error starting Portable_OGCS_v2.7.7

### ISSUE THREAD

**Issue by JPHFS** [phw198/OutlookGoogleCalendarSync#658]
**OGCS Version**: v2.7.7  
**Installed or Portable**: Portable

**Problem Description**  
\ I receive an error message, directly after start the program

**Steps to Reproduce the Issue**  
1. Start OutlookGoogleCalendarSync.exe
1. 

Please see attachment

Thanks!

**Any other information**  
![screenshot - 19 01 25 09_28_38](https://user-images.githubusercontent.com/47025060/51734512-0a79bb80-2085-11e9-82d1-07a9a0e44440.png)
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/2795257/OGcalsync.log)

**Comment by JPHFS** [comment]
The error appears when I try to sync. Using the version 2.7.0, I have the same problem, but when I disable the Option "Description", it works. I believe, that I don't have the right to access the description, BUT in the 2.7.7 version, the description is disabled and I cannot change it (grey). See Attachment
It is something with the rigths. As I just wrote, when I disable every  "What" (like Location), it works in the version 2.7.0, but not at 2.7.7


![screenshot - 19 01 25 11_11_54](https://user-images.githubusercontent.com/47025060/51739600-2e8fc980-2092-11e9-86cc-0474c36a78c5.png)
![screenshot - 19 01 25 11_12_09](https://user-images.githubusercontent.com/47025060/51739611-32bbe700-2092-11e9-8d4e-269a1fb4db6e.png)
![screenshot - 19 01 25 11_12_15](https://user-images.githubusercontent.com/47025060/51739623-36e80480-2092-11e9-8ea2-e4937d578de1.png)

**Comment by phw198** [comment]
Could you upload you logfile v2.7.7 again, after starting OGCS afresh and running the first sync?

Basically, versions after 2.7.0 should be detecting that you do not have rights to sync the Description and therefore disabling that option. Obviously, it shouldn't stop OGCS working entirely!

**Comment by JPHFS** [comment]
Hi,

attached you will find the new log file.

Thanks for your help.

BR,
Joao
________________________________
Von: Paul Woolcock <notifications@github.com>
Gesendet: Montag, 28. Januar 2019 01:06
An: phw198/OutlookGoogleCalendarSync
Cc: JPHFS; Author
Betreff: Re: [phw198/OutlookGoogleCalendarSync] Error starting Portable_OGCS_v2.7.7 (#658)


Could you upload you logfile v2.7.7 again, after starting OGCS afresh and running the first sync?

Basically, versions after 2.7.0 should be detecting that you do not have rights to sync the Description and therefore disabling that option. Obviously, it shouldn't stop OGCS working entirely!

**Comment by JPHFS** [comment]
Sorry, I didn't know that.

Attached you will find the new log.
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/2811231/OGcalsync.log)

**Comment by JPHFS** [comment]
Hi,

Any news about my issue? 

Thanks, 
Joao

**Comment by phw198** [comment]
Hi @JPHFS and apologies for the very delayed reply.

On the up-side, I think this is a duplicate of #702 (yes, I know you reported this first :slightly_smiling_face:) - could you try the hotfix as posted on that issue?
